### DISCUSSION TITLE
Line numbers for code blocks

### DISCUSSION THREAD

**Discussion by sepsol** [RocketChat/Rocket.Chat#31290] [Ideas]
For example in Slack, if you press `Ctrl` + `Shift` + `Enter` you can have code blocks (files) that have line numbers. It is really hard to refer to parts of code in discussions when they don't have line numbers.

**Comment by TonyStone31** [comment]
I have been asking for this and I don't think it is ever gonna happen.

**Comment by reetp** [comment]
It's a NFR.

Open a well documented issue in the correct repo.

https://github.com/RocketChat/feature-requests

**Reply by sepsol** [reply]
✅ Done: https://github.com/RocketChat/feature-requests/issues/914
